src/ros_bridge/Dockerfile

```diff
@@ -1,32 +1,35 @@
 FROM osrf/ros:foxy-desktop
 
-WORKDIR /opt/v-hsc-ros-bridge/
+WORKDIR v-hsc-ros-bridge/
 
 # install ros build tools
 RUN apt-get update && apt-get install -y --no-install-recommends \
       python3-colcon-common-extensions libopencv-dev liblua5.2-dev protobuf-compiler libprotobuf-dev && \
     rm -rf /var/lib/apt/lists/*
 
 # clone ros package repo
-RUN git -C /opt clone \
+RUN git clone \
       -b main \
       https://github.com/robocup-hl-tc/v-hsc-ros-bridge.git
 
 # install ros package dependencies
 RUN apt-get update && \
     rosdep install -y \
       --from-paths \
-        /opt/v-hsc-ros-bridge/src/ros_bridge && \
+        v-hsc-ros-bridge/src/ros_bridge && \
     rm -rf /var/lib/apt/lists/*
 
 # build ros package source
 RUN . /opt/ros/$ROS_DISTRO/setup.sh && \
-    colcon build \
-      --packages-select \
-        ros_bridge
+    cd v-hsc-ros-bridge/ && \
+    colcon build
 
-# source ros package from entrypoint
-RUN sed --in-place --expression \
-      '$isource "$ROS_WS/install/setup.bash"'
+RUN rosdep fix-permissions
```

Comment:
Please add a comment why this comment is necessary. I remember these kinds of commands when I used ROS but I don't remember exactly why this is necessary.

Reply:
That's because I got some permission errors while sourcing setup files. Maybe not happening on other machines.